pkg/synthesis/tests_expected_output/k8s_resources/ExampleExprOrCondsAdmin_AdminPoliciesEnabled/adminPolicies.yaml

```diff
@@ -1,24 +1,63 @@
 apiVersion: policy.networking.k8s.io/v1alpha1
 kind: AdminNetworkPolicy
+metadata:
+    annotations:
+        description: Admin Network Policy To Allow Egress Access To DNS Server
+        nsx-id: egress-dns-rule-id
+    creationTimestamp: null
+    name: egress-dns-policy
+    namespace: default
```

Comment:
```suggestion
```
AdminNetworkPolicy is not a namespaced resource

```
$ kubectl api-resources
NAME                                SHORTNAMES                                      APIVERSION                          NAMESPACED   KIND
adminnetworkpolicies                anp                                             policy.networking.k8s.io/v1alpha1   false        AdminNetworkPolicy
```

Reply:
fixed